pkg/api/util/util.go

```diff
@@ -110,3 +110,22 @@ func ValidateDCARequest(w http.ResponseWriter, r *http.Request) error {
 
 	return err
 }
+
+// IsForbidden returns whether the cluster check runner server is allowed to listen on a given ip
+// The function is a non-secure helper to help avoiding setting an IP that's too permissive.
+// The function doesn't guarantee any security feature
+func IsForbidden(ip string) bool {
+	forbidden := map[string]bool{
+		"":                true,
+		"0.0.0.0":         true,
+		"::":              true,
+		"0:0:0:0:0:0:0:0": true,
```

Comment:
we should probably add stuff like ` “[::]"` here as well?

Reply:
Actually I don't think we need that since it's not a `ip:port` here, and this would result in an invalid listen address (`[[::]]:<port>`).

Reply:
I've added it here just to make it clear when raising the error that's a forbidden address, (not just an invalid address that hasn't been validated by `Listen`)

Reply:
but yes it doesn't make too much sense to check an invalid address, we're adding the `[ ]` anyway so let `Listen` handle that. I'm dropping it. @truthbk let me know if this does make sense to you.